File: project-ewa-master/src/app/model/setting.ts
export class Setting{
  id: number;
  name: string;
  content: string;

  constructor(id: number, name: string, content: string) {
    this.id = id;
    this.name = name;
    this.content = content;
  }

  getName(){
    return this.name;
  }

  getContent(){
    return this.content;
  }
  getId(){
    return this.id;
  }
  setName(_name: string){
    this.name = _name;
  }
  setContent(_content: string){
    this.content= _content;
  }

  static returnObject(settings: Setting) {
    return new Setting(settings.id, settings.name, settings.content);
  }
}

File: project-ewa-master/ewaserver/src/main/java/com/example/ewaserver/entity/Setting.java
package com.example.ewaserver.entity;

import javax.persistence.*;
import javax.validation.constraints.NotNull;


@Entity
//@Table(name = "setting")
public class Setting implements Identifiable {
  @Id
  @NotNull
  @GeneratedValue(strategy = GenerationType.AUTO, generator = "gen_settings")
  @Column(name = "setting_id")
  private long id;
  @Column(name="content")
  private long content;
  @Column(name="name")
  private String name;

  public Setting(){
  }


  public Setting(long content, long id, String name){
    this.id = id;
    this.content = content;
    this.name = name;
  }


  @Override
  public Long getId() {
    return id;
  }

  @Override
  public void setId(Long id) {
    this.id = id;
  }

  public long getContent() {
    return content;
  }

  public void setContent(long content) {
    this.content = content;
  }

  public String getName() {
    return name;
  }

  public void setName(String name) {
    this.name = name;
  }
}

